services/dynamic-scheduler/requirements/_base.in

```diff
@@ -14,9 +14,10 @@
 --requirement ../../../packages/service-library/requirements/_fastapi.in
 
 
-
+aiocache[redis]
 fastapi
 httpx
 packaging
+python-socketio
```

Comment:
I think there is an installation mode to "only" install the necessary stuff for sending messages instead of the whole package. maybe ask @pcrespov for this.